frontend/src/Api.tsx

```diff
@@ -0,0 +1,5 @@
+const api = process.env.REACT_APP_API;
+
+export const API_USER_LOGIN = `${api}/user/login`;
+export const API_USER_FORGOT = `${api}/user/forgot`;
+export const API_USER_RESET = `${api}/user/reset`;
```

Comment:
@phong1233 Tell me what you think of this approach.

Unfortunately, I wasn't able to use the .env file from the env/ folder.  facebook/create-react-app#4283 seems to indicate that you can't select where the .env file is located unless you eject from React.

Reply:
This is how I did it in the backend https://github.com/Talos10/SOEN390-team-9/blob/main/backend/config.ts
The env file is only used when launching using docker

Reply:
Trueeee.
I'll adjust it.